templates/toss/src/skills/toss.ts

```
@@ -0,0 +1,330 @@
+      content: { params },
+      sender,
+    },
+    group,
+    walletService,
+  } = context;
+
+  if (!group) {
+    await context.reply("This command can only be used in a group.");
+    return;
+  }
+  if (params.description && params.options && !isNaN(Number(params.amount))) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)